benches/rustposix-criterion.rs

```diff
@@ -46,14 +36,20 @@ fn get_dummy_cage() -> Cage {
 
 pub fn basic_rustposix_benchmark(c: &mut Criterion) {
 
-    // First, just do a basic benchmark to show the runner works...
-    c.bench_function("fib 20", |b| b.iter(|| fibonacci(black_box(20))));
+    // I'm not setting this up properly because I think I don't need that much
+    // state.  I just want to call system calls.
     let a = get_dummy_cage();
```

Comment:
In our tests we do lindrustinit/finalize to setup and shutdown, this will create an initial cage. That may work here as well unless you're trying to test for a specific state. Not partial to either, just pointing out the difference.